.gitmodules

```diff
@@ -1,9 +1,6 @@
-[submodule "tribler"]
-	path = tribler
-	url = https://github.com/Tribler/tribler.git
 [submodule "docs"]
 	path = docs
 	url = https://git.overleaf.com/5049466xzvfzv
-[submodule "addrgen-master"]
+[submodule "addrgen2"]
 	path = addrgen
```

Comment:
Why is this altered. It was refactored in another branch, which is now inside the master branch, and including this in the pull request will simply undo that refactoring.